src_new/ampManager.js

```diff
@@ -0,0 +1,79 @@
+require("prebid-universal-creative");
+
+window.PWT = window.PWT || {};
+window.pbjs = window.pbjs || {};
+
+var refThis = this;
```

Comment:
remove this if not required now

Reply:
this is required as our function is in PWT function scope. and pbjs is called in an exported function referred by refThis